prism.json

```diff
@@ -0,0 +1,8 @@
+{
+    "PSModules": [
+        {
+            "Name": "VSSetup",
+            "Version": "*"

```

Comment:
Is `Version` always required? It'd be nice if we could just specify a `Name` if we always want the latest version.

Reply:
My bad. I used `*` to get the latest version with then intent to change this to `MAJOR.*` after running `prism install`. I'll change this to `2.*`.